cross-functional/workflow.md

```diff
@@ -1,34 +1,124 @@
-- Area leads are expected to monitor this channel and participate in cross-team conversations as they arise.
-- All 2i2c staff have visibility into the channel and may participate in conversations, but are not expected to do so as part of their daily responsibilities.
+- We expect to see XXX
+- This will be complete when YYY
 
-### Team members expected to monitor this channel
+```[tasklist]
+- [ ] A growing list of tasks and actions used to coordinate work around this initiative.
+- [ ] Each should be an actionable issue that we track independently.
+- [ ] This may be empty when the initiative is created! The initiative's goals should be used to decide what action issues to create next.
+```
+````
+`````
 
-Below is a list of the team members that we **expect** to participate in this channel on a regular basis:
+### Bi-weekly initiatives meetings
 
+There are two bi-weekly meetings with the initiatives team.
+
+- **Initiative planning and refinement**: A combination of **coordination** and **prioritization**. We first walk the board and ensure each initiative has clear next steps, assigned people, and that any coordination between teams is clear. We next zoom out and discuss our overall portfolio of initiatives to ensure we're sequencing them properly. This should inform priorities for sprint planning and delivery.
+- **Initiative retrospectives**: The Initiatives team follows the same retrospective structure we follow for any team. This should consider the results generated from sprint delivery and team retrospectives.
+
+(slack:solution-forum)=
+### The `#initiatives` slack channel
+
+The [`#initiatives` Slack channel](https://2i2c.slack.com/archives/C06G5FAAT63) is a space for cross-organization initiatives and strategic conversation.
+
+### Initiatives team
+
+- @choldgraf : To represent 2i2c's overall strategy and policies
 - @colliand: To represent our overall partnerships strategy and team
 - @jmunroe: To represent our community success efforts
 - @Gman0909 : To represent our product operations and strategy
 - @damianavila : To represent our engineering team's needs
 - @yuvipanda : To represent our technical and open source strategy
-- @choldgraf : To represent 2i2c's overall strategy and policies
 - @haroldcampbell : To represent 2i2c's overall system of work and operations
 - @aprilmj : To represent 2i2c's People Operations and team support
+
+(board:delivery)=
+## Delivery boards
+
+Our delivery boards are where we organize the tasks, deliverables and stories that drive our daily actions and our bi-weekly planning cycles.
+All items on the delivery board should be linked to an `In progress` initiative, or be related to reactive work.
+
+There are two delivery boards:
+
+### Operations board
+
+The [**Operations board**](https://github.com/orgs/2i2c-org/projects/47) is a team-wide board focused around delivery. Includes all organizational tasks/actions except for Engineering.
```

Comment:
> Includes all organizational tasks/actions except for Engineering.

So, we are expecting Product to use this board as well, am I correct?
I am "worried" that a definition of "everything except Eng" is actually hidding the complexities for each area inside that "bucket".

I just want to make sure people realize we are purposelly "hidding" complexity (which might the the right path!)

Btw, this is not a blocker IMHO, just some thoughts I had when I reviewed this PR.

Reply:
Yes, the idea is that we all use it to track issues that are focused on delivery and tactical-level things.

> I am "worried" that a definition of "everything except Eng" is actually hidding the complexities for each area inside that "bucket".

Could you go into more detail on this? What kind of hidden complexity are you talking about?

Reply:
As things specializes (ie. in different areas), it is easier to find differences in definitions, policies, etc.
Even cadences could be different.
With this proposal we are symplifiying pieces, assuming those differences do not exist or are just minimal ones.
We know in advance this is not true, but we can choose to model it like that if we abstract pieces enough high (the current approach). This is what I describe as "hidding"complexity, the fact we know things might be quite different but we are trying to model them with the same common abtraction. And that's a decision we are making and it is totally OK as long as we realize we are taking that decision (and understand their limitations).

Btw, re-reading this above paragraph I notice I am going maybe too meta 😉

Reply:
Gotcha - thanks for this nuance. I think you have a valid concern, and it's something that we should keep in mind once we start following the retrospective process for these boards. That way we can learn and iterate